sdk/monitor/azure-monitor-opentelemetry-exporter/tests/trace/test_trace.py

```diff
@@ -58,7 +58,7 @@ def tearDownClass(cls):
     def test_constructor(self):
         """Test the constructor."""
         exporter = AzureMonitorTraceExporter(
-            connection_string="InstrumentationKey=4321abcd-5678-4efa-8abc-1234567890ab",
+            instrumentation_key="4321abcd-5678-4efa-8abc-1234567890ab",
```

Comment:
We instantiate with connection string, not ikey. Why is this change here?

Reply:
I only did this in relation to your comment earlier about the addition of `instrumentation_key` to the connection string parser. I will update these tests depending on the outcome of that thread. However if we chose to only instantiate via connection string, this test becomes redundant, as we should only support a single way to instantiate with connection string, which is already tested.